### PR TITLE
Ignore parameter values out of range

### PR DESCRIPTION
**Pull request by pnt325**
Correct sensor value has out of range #190

### CODE REVIEW

File: src/Main/utils.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    return 0;
+  }
+  if (value > 10000) {
+    return 10000;
+  }
+  return value;
+}
+
+int utils::correctPMS(int value) {
+  if (value < 10) {
```
Comment (Byter09) [inline src/Main/utils.cpp:42]: Why not allow values below 10? My sensors regularly go below that. Maybe limit it to 0?

Comment (pnt325) [reply src/Main/utils.cpp:42]: @Byter09 Please check the issue #190 and add your comments at there.

Comment (Byter09) [reply src/Main/utils.cpp:42]: I mean I don't have to. Even that issue says 0 should be the lowest value for PM data.

**Review by airgradienthq — CHANGES_REQUESTED** [review]
A few things:
Only negative PM values (below 0) need to be ignored. 0..10 are valid readings and should be allowed.

Also, if a value is outside the range, the value should be completely ignored.

So for example Temperature -100 should NOT return -40 but be ignored. This would apply to all checks.

**Comment by pnt325** [comment]
> A few things: Only negative PM values (below 0) need to be ignored. 0..10 are valid readings and should be allowed.
> 
> Also, if a value is outside the range, the value should be completely ignored.
> 
> So for example Temperature -100 should NOT return -40 but be ignored. This would apply to all checks.

For this way we don't need to correct value in range, just need to verify it's valid or not before use(show on display or sync to cloud). 

The correction method will be remove and replace check valid valid method. 

```c
bool utils::validTemperature(float value) {
  if (value >= VALID_TEMPERATURE_MIN && value <= VALID_TEMPERATURE_MAX) {
    return true;
  }
  return false;
}
```

@airgradienthq  How do you think?

**Comment by airgradienthq** [comment]
@pnt325 yes I think that's how it should work.

**Review by Byter09 — COMMENTED** [review]
I know I was asked to review this, but I don't know internals. So from a basic look at it, it looks good to me. :stuck_out_tongue:

**Review by airgradienthq — APPROVED** [review]

**Pull request merged**